iceoryx2-cli/iox2/Cargo.toml

```diff
@@ -11,8 +11,13 @@ rust-version = { workspace = true }
 version = { workspace = true }
 
 [dependencies]
-human-panic = { workspace = true }
+anyhow = { workspace = true }
 better-panic = { workspace = true }
-clap = { workspace = true }
 colored = { workspace = true }
-thiserror = { workspace = true }
+clap = { workspace = true }
+human-panic = { workspace = true }
+cargo_metadata = { version = "0.18.1" }
```

Comment:
Should this and `tempfile` be moved to the workspace `Cargo.toml`? I left them here since I doubt they will be used elsewhere.

Reply:
Yes. The idea is that the whole workspace has unified and compatible dependencies. Therefore, everything has to be in the workspace and is then used with `workspace = true` by the individual crates of that workspace.